frontend/src/lib/logic/apiStatusLogic.ts

```diff
@@ -46,6 +61,13 @@ export const apiStatusLogic = kea<apiStatusLogicType>([
                     const data = await response?.json()
                     if (data.detail === 'This action requires you to be recently authenticated.') {
                         actions.setTimeSensitiveAuthenticationRequired(true)
+                    } else if (data.code === 'permission_denied') {
```

Comment:
This is not fool proof best it looks at all requests and I only really want to look at specific ones.